engine/common/rpc/convert/collections.go

```diff
@@ -56,6 +56,7 @@ func MessageToLightCollection(m *entities.Collection) (*flow.LightCollection, er
 	}), nil
 }
 
+// FullCollectionToMessage converts a full collection to a slice of protobuf transaction messages
```

Comment:
these methods are a special case for error documentation, where all returned errors are benign. In this case, we document this and do not need to list the sentinels. See 4 in the [coding convention](https://github.com/onflow/flow-go/blob/master/CodingConventions.md#error-handling).

```suggestion
// FullCollectionToMessage converts a full collection to a slice of protobuf transaction messages
//
// All errors indicate the input could not be correctly converted.
```